modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7026,9 +7026,9 @@ hint-for-narrow-portlet=n
 hint-for-wide-portlet=w
 historical-charts=Historical Charts
 history=History
-history[record]=History
 history-count=History Count
 history-count-help=This determines the number of previous passwords to keep in the history.
+history[record]=History
```

Comment:
I think we should rename the commit title from `LPS-138662 buildLang for key history[record]` to `LPS-138662 buildLang`

Reply:
Done!